code/modules/surgery/organs/lungs.dm

```diff
@@ -228,21 +278,21 @@
 
 		if(!(GLOB.mutations_list[COLDRES] in H.dna.mutations) && !(RESISTCOLD in species_traits)) // COLD DAMAGE
 			switch(breath.temperature)
```

Comment:
It's rather peculiar, just by making the below changes, you only ever take heat damage, even when breathing in something too cold. Can't figure this one out. Any ideas @AnturK @Cyberboss